games-action/powermanga/powermanga-0.93.1.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A 2D arcade shoot-em-up game"
+DESCRIPTION="Powermanga is an arcade 2D shoot-em-up game with 41 levels \
+and more than 200 sprites.
+Join POWERMANGA, an elite unit tasked to clear space of pirates, \
+and improve the safety of peaceful voyagers and navigators.
+Collect gems to upgrade your ship (red, green), restore energy (purple) \
+and gain satellite shields (yellow).
+Upgrades include mobility boost, guided missiles, rear fire shot, and more.
+
+Controls:
+ARROW KEYS -> Move spaceship
+SPACE -> Shoot
+CTRL -> Aquire an activated option
+ESC -> Display menu
+P -> Pause the game
+F -> Toggle fullscreen"
+HOMEPAGE="http://linux.tlk.fr/games/Powermanga"
+REVISION="1"
+COPYRIGHT="1998-2019 TLK Games"
+LICENSE="GNU GPL v3"
+SOURCE_URI="https://github.com/brunonymous/powermanga/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="010987a3cb27a1f9388a212f637977692284b5616952efa1efae09d2464e9249"
+SOURCE_DIR="Powermanga-$portVersion"
+PATCHES="powermanga-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Comment:
Add secondary architecture here:
```diff
+SECONDARY_ARCHITECTURES="?x86"
```
It builds for it but crashes, you already have the $secondaryArchSuffix in place, so we could file an issue for that later

Reply:
Should I remove ?x86 from Architectures as well. Line 26.

Reply:
You can, we don't provide a "primary architecture" for x86 anymore, I don't use it in recipes anymore